### PR TITLE
11.6 Runtime Release

### PR DESCRIPTION
**Pull request by lzdanski**
https://astronomer.slack.com/archives/C026XPSJ0JK/p1718982918618459

### CODE REVIEW

File: astro/runtime-release-notes.md [hunk 1]
```diff
@@ -29,6 +29,19 @@ If you're upgrading to receive a specific change, ensure the release note for th
...
 :::
 
+## Astro Runtime 11.6.0
+
+- Release date: June 25, 2024
+- Airflow version: 2.9.2
+
+### Early access Airflow bug fixes
+
+- Fixed a bug by adding `[webserver]update_fab_perms` to the deprecated config list, so that you can now use `[fab]update_fab_perms`, the preferred option, instead ([#40317](https://github.com/apache/airflow/pull/40317))
```
Comment (jedcunningham) [inline astro/runtime-release-notes.md:39]:
```suggestion
- Fixed a bug where FAB config options (e.g. `[fab] update_fab_perms`) were not checking for values in the deprecated webserver config section (e.g. `[webserver] update_fab_perms`) ([#40317](https://github.com/apache/airflow/pull/40317))
```
I think this better reflects what this change actually is. The new one still worked, it was the fallback to the old that was broken.

File: astro/runtime-release-notes.md [hunk 2]
```diff
@@ -29,6 +29,19 @@ If you're upgrading to receive a specific change, ensure the release note for th
 
 :::
 
+## Astro Runtime 11.6.0
+
+- Release date: June 25, 2024
```
Comment (jedcunningham) [inline astro/runtime-release-notes.md:34]: Just a reminder that we'll need to fix this date whenever we do cut the release.

**Review by jedcunningham — APPROVED** [review]

File: astro/runtime-release-notes.md (hunk 2 above)
Comment (utkarsharma2) [inline astro/runtime-release-notes.md:34]:
```suggestion
- Release date: June 28, 2024
```

File: astro/runtime-provider-reference.md [hunk 3]
```diff
@@ -26,6 +26,60 @@ docker run --rm quay.io/astronomer/astro-runtime:11.0.0 pip freeze | grep apache
...
+| apache-airflow-providers-microsoft-azure | 9.0.1     |
+| apache-airflow-providers-mysql           | 5.6.1     |
+| apache-airflow-providers-openlineage     | 1.8.0     |
+| apache-airflow-providers-postgres        | 5.11.1    |
+| apache-airflow-providers-redis           | 3.7.1     |
+| apache-airflow-providers-smtp            | 1.7.1     |
+| apache-airflow-providers-sqlite          | 3.8.1     |
+| astro-sdk-python                         | 1.8.1     |
+| astronomer-providers                     | 1.19.1    |
+| astronomer-providers-logging             | 1.5.1     |
```
Comment (utkarsharma2) [inline astro/runtime-provider-reference.md:62]:
```suggestion
| package_name                             | version   |
|:-----------------------------------------|:----------|
| apache-airflow-providers-amazon          | 8.25.0    |
| apache-airflow-providers-celery          | 3.7.2     |
| apache-airflow-providers-cncf-kubernetes | 8.3.2     |
| apache-airflow-providers-common-io       | 1.3.2     |
| apache-airflow-providers-common-sql      | 1.14.1    |
| apache-airflow-providers-datadog         | 3.6.1     |
| apache-airflow-providers-elasticsearch   | 5.4.1     |
| apache-airflow-providers-fab             | 1.2.0     |
| apache-airflow-providers-ftp             | 3.10.0    |
| apache-airflow-providers-google          | 10.19.0   |
| apache-airflow-providers-http            | 4.12.0    |
| apache-airflow-providers-imap            | 3.6.1     |
| apache-airflow-providers-microsoft-azure | 9.0.1     |
| apache-airflow-providers-mysql           | 5.6.2     |
| apache-airflow-providers-openlineage     | 1.8.0     |
| apache-airflow-providers-postgres        | 5.11.2    |
| apache-airflow-providers-redis           | 3.7.1     |
| apache-airflow-providers-smtp            | 1.7.1     |
| apache-airflow-providers-sqlite          | 3.8.1     |
| astro-sdk-python                         | 1.8.1     |
| astronomer-providers                     | 1.19.1    |
| astronomer-providers-logging             | 1.5.1     |
```

Comment (utkarsharma2) [inline astro/runtime-provider-reference.md:62]:
```suggestion
| apache-airflow-providers-amazon          | 8.25.0    |
| apache-airflow-providers-celery          | 3.7.2     |
| apache-airflow-providers-cncf-kubernetes | 8.3.2     |
| apache-airflow-providers-common-io       | 1.3.2     |
| apache-airflow-providers-common-sql      | 1.14.1    |
| apache-airflow-providers-datadog         | 3.6.1     |
| apache-airflow-providers-elasticsearch   | 5.4.1     |
| apache-airflow-providers-fab             | 1.2.0     |
| apache-airflow-providers-ftp             | 3.10.0    |
| apache-airflow-providers-google          | 10.19.0   |
| apache-airflow-providers-http            | 4.12.0    |
| apache-airflow-providers-imap            | 3.6.1     |
| apache-airflow-providers-microsoft-azure | 9.0.1     |
| apache-airflow-providers-mysql           | 5.6.2     |
| apache-airflow-providers-openlineage     | 1.8.0     |
| apache-airflow-providers-postgres        | 5.11.2    |
| apache-airflow-providers-redis           | 3.7.1     |
| apache-airflow-providers-smtp            | 1.7.1     |
| apache-airflow-providers-sqlite          | 3.8.1     |
| astro-sdk-python                         | 1.8.1     |
| astronomer-providers                     | 1.19.1    |
| astronomer-providers-logging             | 1.5.1     |
```

File: astro/runtime-provider-reference.md [hunk 4]
```diff
@@ -26,6 +26,60 @@ docker run --rm quay.io/astronomer/astro-runtime:11.0.0 pip freeze | grep apache
...
+| apache-airflow-providers-elasticsearch | 5.4.1     |
+| apache-airflow-providers-fab           | 1.1.1     |
+| apache-airflow-providers-ftp           | 3.9.1     |
+| apache-airflow-providers-http          | 4.11.1    |
+| apache-airflow-providers-imap          | 3.6.1     |
+| apache-airflow-providers-mysql         | 5.6.1     |
+| apache-airflow-providers-postgres      | 5.11.1    |
+| apache-airflow-providers-smtp          | 1.7.1     |
+| apache-airflow-providers-sqlite        | 3.8.1     |
+| astronomer-providers-logging           | 1.5.1     |
```
Comment (utkarsharma2) [inline astro/runtime-provider-reference.md:79]:
```suggestion
| apache-airflow-providers-celery        | 3.7.2     |
| apache-airflow-providers-common-io     | 1.3.2     |
| apache-airflow-providers-common-sql    | 1.14.1    |
| apache-airflow-providers-elasticsearch | 5.4.1     |
| apache-airflow-providers-fab           | 1.2.0     |
| apache-airflow-providers-ftp           | 3.10.0    |
| apache-airflow-providers-http          | 4.12.0    |
| apache-airflow-providers-imap          | 3.6.1     |
| apache-airflow-providers-mysql         | 5.6.2     |
| apache-airflow-providers-postgres      | 5.11.2    |
| apache-airflow-providers-smtp          | 1.7.1     |
| apache-airflow-providers-sqlite        | 3.8.1     |
| astronomer-providers-logging           | 1.5.1     |
```

**Review by utkarsharma2 — APPROVED** [review]

File: astro/runtime-release-notes.md [hunk 5]
```diff
@@ -29,6 +29,19 @@ If you're upgrading to receive a specific change, ensure the release note for th
...
 :::
 
+## Astro Runtime 11.6.0
+
+- Release date: June 28, 2024
+- Airflow version: 2.9.2
+
+### Early access Airflow bug fixes
+
+- Fixed a bug where FAB config options, such as `[fab] update_fab_perms`, were not checking for values in the deprecated webserver config section, for example, `[webserver] update_fab_perms` ([#40317](https://github.com/apache/airflow/pull/40317))
```
Comment (paolaperaza) [inline astro/runtime-release-notes.md:39]:
```suggestion
- Fixed a bug where FAB config options, such as `[fab] update_fab_perms`, were not checking for values in the deprecated webserver config section. For example, `[webserver] update_fab_perms` ([#40317](https://github.com/apache/airflow/pull/40317))
```

**Review by paolaperaza — APPROVED** [review]

**Pull request merged**